pkg/operator/controller/controller_cabundle_configmap.go

```diff
@@ -64,14 +64,18 @@ func (r *reconciler) ensureCABundleConfigMaps(dns *operatorv1.DNS) error {
 		}
 		haveSource, source, err := r.currentCABundleConfigMap(sourceName)
 		if err != nil {
-			errs = append(errs, fmt.Errorf("failed to get source ca bundle config map %s: %w", sourceName.Name, err))
+			errs = append(errs, fmt.Errorf("failed to get source ca bundle configmap %s: %w", sourceName.Name, err))
+			continue
+		}
+		if !haveSource {
```

Comment:
If we log it for the source, why don't we log it for the destination below?

Reply:
The source cm is provided by the user, whereas the destination is created by the operator based on the source cm. If the destination cm does not exist, that does not indicate a problem as it will be created by the operator later in line 99.

Reply:
https://github.com/openshift/cluster-dns-operator/pull/321/files#diff-6321af060e023fc35bbfc959a1073934fbd15ff8778cc24126d9d4470e87ae77R152-R154 shows that if there is no source, then `want` is false.  If `want` and `have` are both false, we don't create anything, correct?